src/orion/algo/space/configspace.py

```diff
@@ -185,6 +187,13 @@ def _from_categorical(dim: CategoricalHyperparameter) -> Categorical:
     return Categorical(dim.name, choices)
 
 
+@to_oriondim.register(OrdinalHyperparameter)
+def _from_ordinal(dim: OrdinalHyperparameter) -> Categorical:
```

Comment:
Is ordinal best mapped to categorical or integer? Categorical is loosing the importance of the ordering.

Reply:
ordinal values does not have to be `int`. https://automl.github.io/ConfigSpace/main/api/hyperparameters.html#ConfigSpace.hyperparameters.OrdinalHyperparameter